packages/design/jobberStyle.js

```diff
@@ -171,8 +171,33 @@ function getResolvedSCSSVariables(cssProperties) {
     } else if (isSizeVariable) {
       const suffix = customProperties["--" + cssVar].includes("%") ? "%" : "px";
       return [...acc, `$${cssVar}: ${resolvedCssVars[cssVar]}${suffix};`];
+    } else if (cssVar.includes("shadow")) {
+      const resolvedShadowProperty = resolveShadow(
+        customProperties["--" + cssVar],
+      );
+
+      return [...acc, `$${cssVar}: ${resolvedShadowProperty};`];
     } else {
       return acc;
     }
   }, []);
 }
+
+function resolveShadow(shadowValue) {
+  const splittedValue = shadowValue.split(" ").filter(n => n);
```

Comment:
Really minor wording suggestion on the naming

```suggestion
  const splitValue = shadowValue.split(" ").filter(n => n);
```

Reply:
Done! Thank you for the suggestion